### PR TITLE
feat: Tweaks to flows 

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://www.notion.so/zerotech/dApp-Projects-e96437225e264ae8ae8d46ca5f4d7b35?p=698952d5f8974461b07ee14a71b1b1e6)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
- [x] Feature

## 3. What is the old behaviour?
- Some UI tweaks to existing flows.
- Prevent user from entering own address for transferring NFT

## 4. What is the new behaviour?
- UI updates.
- User can no longer enter their own address when transferring NFT. Error states have been added.

Example of a couple of changes:
https://www.loom.com/share/1dd1103a60fc4cfa862ccd994ad99ccf

### CODE REVIEW

**Comment by domw30** [comment]
This branch has been created from ```feat/dom/accept-bid-sdk```. The files changed shall reduce and up to date once this branch has been merged.

**Comment by domw30** [comment]
@padala, ready for review. A couple of changes here to transfer ownership. Not sure if we want to merge this (after approval) to ```develop``` or ```feat/subbu/sdk-integration```. Let me know if you have a suggestion. Thanks!

**Comment by padala** [comment]
@domw30  Sorry for the confusion. We don't have to delete the `RequestsTable` and associated files. It is not being used but we might use them in the future again. 
Can you please revert the deleted files? I already reverted those in the main branch.

**Review by padala — APPROVED** [review]

**Pull request merged**
